public/app/plugins/datasource/elasticsearch/datasource.test.ts

```diff
@@ -701,7 +700,7 @@ describe('ElasticDatasource', () => {
       const { ds } = getTestContext({
         data,
         database: 'genuine.es7._mapping.response',
-        jsonData: { esVersion: '7.10.0' },
+        jsonData: {},
```

Comment:
@gabor Can we please remove all of these empty `jsonData` as we don;t use it to override anything and it is not required?